spec/features/responsive/home_page_responsive_spec.rb

```diff
@@ -6,20 +6,20 @@
       visit root_path
     end
     it "should show the facets on large screens" do
-      within(".blacklight-format_main_ssim") do
-        expect(page).to have_css(".panel-title", text: "Resource type")
+      within(".blacklight-access_facet") do
+        expect(page).to have_css(".panel-title", text: "Access")
         within("ul.facet-values") do
-          expect(page).to have_css("li a", text: "Book", visible: true)
-          expect(page).to have_css("li a", text: "Newspaper", visible: true)
+          expect(page).to have_css("li a", text: "Online", visible: true)
+          expect(page).to have_css("li a", text: "At the Library", visible: true)
         end
       end
 
       page.driver.resize("700", "700")
 
-      within(".blacklight-format_main_ssim") do
-        expect(page).to have_css(".panel-title", text: "Resource type")
-        expect(page).not_to have_css("li a", text: "Book", visible: true)
-        expect(page).not_to have_css("li a", text: "Newspaper", visible: true)
+      within(".blacklight-access_facet") do
+        expect(page).to have_css(".panel-title", text: "Access")
+        expect(page).not_to have_css("li a", text: "Online", visible: true)
+        expect(page).not_to have_css("li a", text: "At the Library", visible: true)
       end
     end
   end
```

Comment:
I would recommend doing something like `git reset HEAD~` to unstage the changes in your commit, and then checkout the files that do not have anything to do with your PR.

From a quick glance, it looks like there are unrelated changes in `_home_page.html.erb`, `facets_helper.rb`, and this file `home_page_responsive_spec.rb`.

If that doesn't work right off the bat, let me know and I'll see what I can do to help.

The main reason I'm being such a stickler about this is, if we need to revert this commit down the road we would inadvertently revert adding the language facet to the home page (which would be unexpected to say the least).